go.mod

```diff
@@ -40,6 +40,5 @@ require (
 	github.com/stretchr/testify v1.3.0
```

Comment:
this seems like an unrelated change?

Reply:
This is done by the `go mod tidy` command in `make presubmit` automatically.